### ISSUE TITLE
[feature request]QT6-based file picker - xdg-desktop-portal-gtk drop-in replacement

### ISSUE THREAD

**Issue by izmyname** [hyprwm/xdg-desktop-portal-hyprland#244]
Having drop-in replacement for xdg-desktop-portal-gtk, based on qt6 would be great, also it'll fit nicely into overall qt6-based hypr* experience. Especially, since we've got qt6-based hyprpolkit.

**Comment by vaxerski** [comment]
you can also use the kde portal though its notoriously wonky, or the lxqt one (though that one is like qt4 or 5)

**Comment by izmyname** [comment]
Yep, but that's the point of why having hyprland's own qt file picker would be nice - any other option is suck, like with the polkits.

**Comment by izmyname** [comment]
I think, with polkit being a thing, a file picker is a matter of the time.

**Comment by vaxerski** [comment]
prolly not

**Comment by izmyname** [comment]
:c

**Comment by izmyname** [comment]
Closing as no more relevant

**Comment by izmyname** [comment]
I guess I'll reopen it, since the idea of having Hyprland's own file picker is too good and fits too well into the rest hypr* workflow to not mention it. Feel free to close, if there is no chance we'd get one, though.
